legal-api/src/legal_api/reports/report.py

```diff
@@ -960,8 +966,8 @@ def _format_continuation_in_data(self, filing):
         region = None
         if region_code and region_code.upper() != 'FEDERAL':
             region = pycountry.subdivisions.get(code=f'{country_code}-{region_code}')
-            filing['foreignJurisdiction']['region'] = region.name.upper() if region_code else ''
-        filing['foreignJurisdiction']['country'] = country.name.upper()
+            filing['foreignJurisdiction']['region'] = region.name if region_code else ''
```

Comment:
This is to match the design:
![image](https://github.com/user-attachments/assets/9e55fbc9-7cab-4e2a-bc16-70f95b2bd32b)